src/main.rs

```diff
@@ -124,7 +107,7 @@ fn main() {
 
         let mut last_frame = Instant::now();
 
-        let target_fps = 30;
+        let target_fps = 240;
```

Comment:
This is super fast! Don't forget to try `$ cargo run --release` to build with optimizations.